src/Serialization/main.jl

```diff
@@ -26,6 +25,17 @@ end
 
 const backref_sym = Symbol("#backref")
 
+@Base.kwdef struct MetaData
+    author::Union{String, Vector{String}, Nothing} = nothing
+    name::Union{String, Nothing} = nothing
+    email::Union{String, Nothing} = nothing
+    affiliation::Union{String, Nothing} = nothing
```

Comment:
I propose to condense this into
```suggestion
    author_orcid::Union{String, Nothing} = nothing
    name::Union{String, Nothing} = nothing
```
The reason is that email and affiliation often change for researchers, while the orcid stays constant. Most researchers have an orcid nowadays and otherwise it's not hard to set up. If we kept email and affiliation we will very fast have data where the author cannot be contacted.